pages/business/setup/signup.tsx

```diff
@@ -46,9 +51,7 @@ const SetupSignupPage: React.FC<WithOwnerProps> = () => {
   ) => {
     try {
       setLoading(true)
-      const affiliate = isCareEnv
-        ? 'care'
-        : localStorage.getItem('rcvr_affiliate')
+      const affiliate = getAffiliate()
```

Comment:
```suggestion
      const affiliate = localStorage.getItem('rcvr_affiliate')
```
This was used to filter users before we introduced the Frontend-Model. We should revert this override logic.

Reply:
I don't get it.

intro.tsx
```
  React.useEffect(() => {
    if (query.affiliate) {
      localStorage.setItem('rcvr_affiliate', query.affiliate.toString())
    }
  }, [query.affiliate])
```

Where is this affiliate thing being set? I checked the registrationflow via getrecover.de and www.recovercare.de and nothing is setting the url_parameter needed to set this localstorage item. As I see it every new registration would be set to null then. Is this really what is needed?

Reply:
It is set manually when marketing provides that to partners ex: https://rcvr.app/business/setup/intro?affiliate=partner_name